core/templates/components/summary-tile/learner-story-summary-tile.component.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for a canonical story tile.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { TopicViewerDomainConstants } from 'domain/topic_viewer/topic-viewer-domain.constants';
+import { Input } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { AssetsBackendApiService } from 'services/assets-backend-api.service';
+import { AppConstants } from 'app.constants';
+import { StorySummary } from 'domain/story/story-summary.model';
+
+@Component({
+  selector: 'oppia-learner-story-summary-tile',
```

Comment:
Where is this component used?

Reply:
It is used in the [Current Goals](https://xd.adobe.com/view/b54892aa-dbe6-49cf-bc4f-9bb8c77de619-e01b/screen/992e8f3c-ebcc-4a6e-991a-b85ffb0749e4/specs/) section of the Goals tab to display the story card. This component will also be used in the [Stories Completed ](https://xd.adobe.com/view/b54892aa-dbe6-49cf-bc4f-9bb8c77de619-e01b/screen/17254418-d5e8-4ff0-aa7c-31cbe204f45c/specs/)section of the Progress tab and the [Continue where you left off section](https://xd.adobe.com/view/b54892aa-dbe6-49cf-bc4f-9bb8c77de619-e01b/specs/) of the Home tab.